packages/flame/lib/src/components/sprite_component.dart

```diff
@@ -122,6 +127,7 @@ class SpriteComponent extends PositionComponent with HasPaint {
       canvas,
       size: size,
       overridePaint: paint,
+      bleed: bleed,
```

Comment:
why do we set it in the constructor of sprite but also pass it here? is that duplicated?

Reply:
yeah, this is not needed anymore, good catch.

Reply:
Actually, it is not duplicated, this isn't passed in the constructor of the Sprite, just the component.